### ISSUE TITLE
loss=nan

### ISSUE THREAD

**Issue by FLHonker** [qjy981010/CRNN.IIIT-5K.pytorch#5]
未改动源码，但是loss一致增加，最后为nan，无法正确训练。

**Comment by qjy981010** [comment]
试试把lr调小10倍，一般就好了。或者加个clip_grad试试

**Comment by FLHonker** [comment]
感谢，回复真及时！

**Comment by FLHonker** [comment]
还是不行啊，只是延缓了loss上升，但还是不下降，最终nan。

**Comment by qjy981010** [comment]
```
loss.backward()
torch.nn.utils.clip_grad_norm(model.parameters(), clipping_value)
optimizer.step()
```
试试clip_grad？我这里没有环境了，不方便跑代码

**Comment by FLHonker** [comment]
不行，loss在6-12左右，根本不下降

**Comment by qjy981010** [comment]
我配了个环境试着没问题啊，你用的别的数据集？

**Comment by FLHonker** [comment]
麻烦你了。用的就是IIT-5K，我用的pytorch1.2啊，应该没毛病

**Comment by FLHonker** [comment]
我就改成了使用cuda，难道你使用cpu没问题？我改成GPU就不行？

**Comment by qjy981010** [comment]
我用的也是gpu，pytorch1.0，训了60个epoch，loss大概一点几，测试集上有35%的acc。
再调小lr继续训的话loss还能降。
你要不试试pytorch1.0吧

**Comment by FLHonker** [comment]
好，谢谢！
